net/adguardhome/Makefile

```
@@ -6,13 +6,14 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=adguardhome
-PKG_VERSION:=0.107.36
+PKG_VERSION:=0.107.40
```

```suggestion
PKG_VERSION:=0.107.41
```
might as well bump up one more; PKG_MIRROR_HASH will also need updating in that case

---

Done.